front/src/store/tv/__tests__/actions.test.ts

```diff
@@ -0,0 +1,14 @@
+import { SAVE_SEARCH_QUERY, saveSearchQueryAction } from '../actions';
```

Comment:
Would it be possible to use the extension `actions.unit.ts`?

Reply:
normally the convention is <smth>.test.ts or <smth>.spec.test and that is what we told the jest to look for

Reply:
Ok test.ts is good 👍